rosplan_interface_mapping does not build correctly at first attempt

at the moment building 2 times is needed, due to a implementation bug, the following error shows in terminal:

```
Errors     << rosplan_interface_mapping:make /home/oscar/ros_ws/build_test_ws/logs/rosplan_interface_mapping/build.make.000.log                                          
In file included from /home/oscar/ros_ws/build_test_ws/src/rosplan_demos/rosplan_demos_interfaces/rosplan_interface_mapping/include/rosplan_interface_mapping/RPRoadmapServer.h:38:0,
                 from /home/oscar/ros_ws/build_test_ws/src/rosplan_demos/rosplan_demos_interfaces/rosplan_interface_mapping/src/RPRoadmapServer.cpp:19:
/home/oscar/ros_ws/build_test_ws/devel/.private/rosplan_interface_mapping/include/rosplan_interface_mapping/AddWaypoint.h:22:9: error: ‘AddWaypointResponse’ does not name a type
 typedef AddWaypointResponse Response;
         ^
/home/oscar/ros_ws/build_test_ws/devel/.private/rosplan_interface_mapping/include/rosplan_interface_mapping/AddWaypoint.h:24:1: error: ‘Response’ does not name a type
 Response response;
```

--- a/rosplan_demos_interfaces/rosplan_interface_mapping/CMakeLists.txt
+++ b/rosplan_demos_interfaces/rosplan_interface_mapping/CMakeLists.txt
@@ -42,6 +42,7 @@ include_directories(
 
 # waypoint generation server
 add_executable(rp_roadmap_server src/RPRoadmapServer.cpp)
+add_dependencies(rp_roadmap_server rosplan_interface_mapping_generate_messages_cpp)
 target_link_libraries(rp_roadmap_server ${catkin_LIBRARIES})
 
 ##########

